gobblin-service/src/test/java/org/apache/gobblin/service/monitoring/JobStatusRetrieverTest.java

```diff
@@ -48,15 +48,20 @@
   abstract void setUp() throws Exception;
 
   protected void addJobStatusToStateStore(Long flowExecutionId, String jobName, String status) throws IOException {
-    addJobStatusToStateStore(flowExecutionId, jobName, status, 0, 0);
+    addJobStatusToStateStore(flowExecutionId, jobName, status, 0, 0, -1);
   }
 
   protected void addJobStatusToStateStore(Long flowExecutionId, String jobName, String status, long startTime, long endTime) throws IOException {
+    addJobStatusToStateStore(flowExecutionId, jobName, status, startTime, endTime, -1);
+  }
+
+  protected void addJobStatusToStateStore(Long flowExecutionId, String jobName, String status, long startTime, long endTime, long timestamp) throws IOException {
```

Comment:
Do we call this overload from non-test code? Looks like timestamp will be -1 all the time in production.